src/components/viz-pane/renderer.tsx

```diff
@@ -69,8 +69,11 @@ export default class VizPane extends React.Component<Props> {
         onChange={this.handleChange}
         paneStyle={{ display: 'flex' }}
         onDragFinished={() => {
-          if (this.props.debugPaneSize === 25) {
+          if (this.props.debugPaneSize === LAYOUT.MinPaneSize) {
```

Comment:
Why is this === and not <?

Reply:
Since , minSize is 25 , I don't think it'll ever go below 25